### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 41

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#229] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255340-036bfac5-b6dc-4cd6-870b-f0527439cdae.png)

**Comment by i0nut02** [comment]
mariorossi
1.  10! / 2!*2!*2!2!
2. 10! / 2!*2!*2!2! - 9!/ 2!*2!*2!2!
3. 8! / 2!*2!*2!2! + 8! / 2!*2!*2!2! - 0

**Comment by neoSnakex34** [comment]
![image](https://user-images.githubusercontent.com/97003970/152638223-65c8cade-3865-4913-ba44-2f050a7bc6cf.png)
con dubbi sul punto 3

**Comment by lord-marcel** [comment]
![esercizio41_page-0001](https://user-images.githubusercontent.com/36630876/152648456-319d8ea4-6df0-4039-abfe-974b62dc99c7.jpg)
